### PR TITLE
SpidermiR R/Bioconductor package

### PR DESCRIPTION
**Pull request by fabio-cumbo**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

The aims of SpidermiR are: i) facilitate the network open-access data retrieval from GeneMania data, ii) prepare the data using the appropriate gene nomenclature, iii) integration of miRNA data in a specific network, iv) provide different standard analyses and v) allow the user to visualize the results. In more detail, the package provides multiple methods for query, prepare and download network data (GeneMania), and the integration with validated and predicted miRNA data (mirWalk, miR2Disease, miRTar, miRTarBase, miRandola, Pharmaco-miR, DIANA, Miranda, PicTar and TargetScan) and the use of standard analysis (igraph) and visualization methods (networkD3).

### CODE REVIEW

File: recipes/bioconductor-spidermir/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+$R CMD INSTALL --build .
+#
+# # Add more build steps here, if they are necessary.
+#
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build
+# process.
+#
```
Comment (daler) [inline recipes/bioconductor-spidermir/build.sh]: Remove extra comments from here please?

**Comment by lecorguille** [comment]
The PR waiting for https://github.com/bioconda/bioconda-recipes/pull/5401, https://github.com/bioconda/bioconda-recipes/pull/5402 and https://github.com/bioconda/bioconda-recipes/pull/5403

**Comment by daler** [comment]
@fabio-cumbo I missed the inter-dependencies across these PRs. Our build system actually allows you to bundle multiple interdependent recipes into the same PR, and they will be built in the right order. This  could save you some time in the future by only having to manage one PR.

**Comment by fabio-cumbo** [comment]
Thanks for the info @daler

**Comment by daler** [comment]
@fabio-cumbo I bundled your changes in #5401, #5402, #5403, and #5404 together into #5478 so hopefully testing and merging will be more straightforward. Closing this for now . . .

**Pull request closed without merging**